apidoc/Titanium/UI/Clipboard/Clipboard.yml

```diff
@@ -153,6 +153,49 @@ methods:
     osver: {ios: {min: "10.0"}}
     platforms: [iphone, ipad]
 
+  - name: remove
+    summary: Removes the clipboard.
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+
+properties:
+  - name: name
+    summary: Create a new named clipboard.
+    description: Create a new clipboard with a given name. See the example on how to use this.
+    parameters:
+      - name: name
+        summary: name of clipboard.
+        type: String
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+    availability: creation
+
+  - name: unique
+    summary: Create a new clipboard identified by a unique system-generated name.
+    description: Create a new clipboard identified by a unique system-generated name. See the example 
+    on how to use this.
+    parameters:
+      - name: unique
+        summary: value of unique.
+        type: Boolean
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+    availability: creation
+
+  - name: allowCreation
+    summary: Create a clipboard identified by name if it doesn,t exist.
+    description: |
+        If create is false, it will return the existing named clipboard but will not create new one.
```

Comment:
Please state that "create" is a property. Either by using `<Titanium.UI.Clipboard.create>` or just using `"create"`. Same in the sentence afterwards.